pallets/ajuna-awesome-avatars/src/types/avatar/versions/v2/avatar_utils.rs

```diff
@@ -1076,6 +1080,18 @@ where
 		(match_count > 0 && (((match_count * 2) + mirror_count) >= 6)).then_some(matches)
 	}
 
+	/// Matches two progress arrays, using `array_2` to upgrade `array_1`.
+	///
+	/// This method return the indexes that matched and the indexes that are considered mirrors.
+	///
+	/// * Matches are indexes that can be increased to the next rarity level.
+	/// * Mirrors are indexes that have been maxed and are of the same variation between both
+	///   arrays.
```

Comment:
I don't think the mirrors have to be maxed, are they? I thought that mirrors are when the rarity level of the `progressive`-byte is at higher or equal to the `sacrifices`'s rarity level?

Reply:
You are right, I'll adjust the comments